.github/workflows/build_tag.yml

```diff
@@ -85,14 +85,12 @@ jobs:
           antrea/antrea-agent-ubuntu-arm:"${DOCKER_TAG}" \
           antrea/antrea-agent-ubuntu-amd64:"${DOCKER_TAG}"
         docker manifest push --purge antrea/antrea-agent-ubuntu:"${DOCKER_TAG}"
-    - name: Trigger Antrea arm tests
+    # Same repository workflow dispatch to run conformance tests
+    - name: Run conformance tests
       uses: benc-uk/workflow-dispatch@v1
       with:
-        repo: vmware-tanzu/antrea-build-infra
-        ref: refs/heads/main
-        workflow: Test Antrea ARM images
-        token: ${{ secrets.ANTREA_BUILD_INFRA_WORKFLOW_DISPATCH_PAT }}
-        inputs: ${{ format('{{ "antrea-repository":"antrea-io/antrea", "antrea-ref":"{0}", "docker-tag":"{1}" }}', github.ref, env.DOCKER_TAG) }}
+        workflow: .github/workflows/conformance.yml
+        inputs: ${{ format('{{ "antrea-version":"{0}", "antrea-image-distro":"ubuntu", "test-suite":"conformance", "runner":"{1}", "antrea-image-platform":"{2}", "always-upload-logs":true }}', github.ref, matrix.runner, matrix.platform) }}
```

Comment:
We can update the kind version to 0.25 as well. Since the default conformance test without given K8s version will use the default one as the kind's default. Now we are still using kind 0.23 and the conformance test image be 1.30 instead of 1.31.2.

Reply:
I will leave that to a future PR, given that `ci/kind/version` should be updated and it will affect other workflows